packages/main/src/ResponsivePopover.ts

```diff
@@ -124,12 +123,17 @@ class ResponsivePopover extends Popover {
 	 * @async
```

Comment:
Something we did not mentioned, but for public methods, we need to add also some new stuff (tags "method", "name" as shown below) to make them available, because I assume if you check the api.json, the method is not there.

```js
/**
	 * Shows popover on desktop and dialog on mobile.
	 * @param {HTMLElement} opener the element that the popover is shown at
	 * @param {boolean} preventInitialFocus Prevents applying the focus inside the popup
	 * @public
	 * @method
	 * @name sap.ui.webc.main.ResponsivePopover#showAt
	 * @async
* @returns {Promise} Resolves when the responsive popover is open
	 */
```